wolox-books/src/app/components/auth/auth.component.html

```diff
@@ -0,0 +1,4 @@
+<nav class="container-nav">
+  <img class="logo-wolox" alt="logo-wolox" src="assets/LogoWolox-Original.png">
+  <span class="text-logout" (click)="logout()">Logout</span>
```

Comment:
If the tag is going to have a click action, which tag is the one you're supposed to use?

Reply:
Which tag are you supposed to use? do you think a span should have a click action?